plugins/vlc-video/vlc-video-source.c

```diff
@@ -520,13 +528,13 @@ static void vlcs_update(void *data, obs_data_t *settings)
 				dstr_copy(&dir_path, path);
 				dstr_cat_ch(&dir_path, '/');
 				dstr_cat(&dir_path, ent->d_name);
-				add_file(c, &new_files.da, dir_path.array);
+				add_file(c, &new_files.da, dir_path.array, network_caching);
```

Comment:
This line goes over the 80 character limit. Insert a line break after "dir_path.array," (right after the comma), delete the proceeding space character (right before "network_caching"), and insert two tabs to indent.